common/component/kafka/kafka.go

```diff
@@ -166,6 +223,128 @@ func (k *Kafka) Close() (err error) {
 	return err
 }
 
+func getSchemaSubject(topic string) string {
+	// For now assumes that subject is named after topic (e.g. `my-topic-value`)
+	return fmt.Sprintf("%s-value", topic)
+}
+
+func (k *Kafka) DeserializeValue(message *sarama.ConsumerMessage, config SubscriptionHandlerConfig) ([]byte, error) {
+	if config.ValueSchemaType == Avro {
```

Comment:
Nit: please use a switch statement here, it just reads better :) 

Alternatively, change this to be short-circuited, such as :

```go
if config.ValueSchemaType != Avro {
    return message.Value, nil
}
```